openforcefield/data/forcefield/tip3p.ffxml

```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='ASCII'?>
+<SMIRNOFF version="0.1">
+  <!-- SMIRks Native Open Force Field (SMIRNOFF) file -->
+  <Date>2017-04-29</Date>
+  <Author>J. D. Chodera, MSKCC; A. Rizzi, Weill Cornell; C. C. Bannan, UC Irvine</Author>
+  <!-- SMIRNOFF file implementing TIP3P water model. -->
+  <NonbondedForce coulomb14scale="0.833333" lj14scale="0.5" sigma_unit="nanometers" epsilon_unit="kilojoules_per_mole">
+    <!-- TIP3P water oxygen with charge override -->
```

Comment:
Yeah, I didn't know what to do about them (see my question in 2nd bullet point above). The charges are ignored currently, but I left them there mainly to be sure they'll be there when we'll implement #25 .

I don't have a strong opinion about them, but writing a comment warning that charges are not working until that issue is tackled sounds like a good idea!